builder/amazon/common/ami_config_test.go

```diff
@@ -30,6 +30,13 @@ func TestAMIConfigPrepare_regions(t *testing.T) {
 		t.Fatalf("shouldn't have err: %s", err)
 	}
 
+	c.AMIRegions = []string{"ap-northeast-1", "ap-northeast-2", "ap-southeast-1",
```

Comment:
You're correct, but I intentionally did not reuse - to make the test fail first :)

For the record - the post commit is 0cad32c69630910a4f42b5c5a0c8d2780503528f